qualang_tools/addons/calibration/README.md

```diff
@@ -0,0 +1,167 @@
+* [Ramsey](#ramsey-measurement): performs a Ramsey measurement where the idle time and pulse frequency can be scanned.
+
+## Usage Example
+1. First the user needs to specify the relevant configuration parameters that will be used for the calibrations.
+Are resuired the _qubit_ and _readout-resonator_ elements and operations, the _integration weights_ and OPX inputs (defined as elements outputs).
+An optional _flux_line_ element and operation can be specified in the case of flux tunable transmon to find the flus soft spot with the resonator spectroscopy.
+Several options are also available for customizing the plots. All of them are written in the snippet below.
+
+2. Then the calibration API is initialized with these parameters. The standard configuration also needs to be provided.
+3. The sequence of calibrations needs to be set using the `.set_` method. All the avainlable calibrations and setting methods are detailed in the *Available calibrations* section below. Note that several calibrations can be set and executed sequentially.
+4. One needs to open a quantum machine manager with the correct IP address.
+5. The calibration can be simulated to check the validity of the sequences. This will plot the analog and digital pulses played during the sequences.
+6. After opening a quantum machine with the configuration, the calibrations can be executed. The flag called `plot` can be take three values:
+   * "full" to plot the results at the end of the sequence.
+   * "live" to plot the results while the sequence is being executed.
+   * None which will not plot anything.
+7. The results are stored in a dictionary accessible under `my_calib.results`.
+
+
+```python
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qualang_tools.addons.calibration.calibrations_API import *
+from configuration import * 
+
+# Relevant configuration parameters
+#############################################################
+resonator_element = "resonator"
+resonator_operation = "readout"
+qubit_element = "qubit"
+qubit_operation = "cw"
+flux_line_element = None
+flux_line_operation = None
+int_w = ["cos", "sin", "minus_sin", "cos"]
+outs = ["out1", "out2"]
+# Options for plotting figures
+options = {
+    "fontsize": 14,
+    "color": "b",
+    "marker": ".",
+    "linewidth": 1,
+    "figsize": (12, 15),
+}
+#############################################################
+
+# Initialize the calibration API with the relevant configuration parameters
+my_calib = QUA_calibrations(
+    configuration=config,
+    readout=(resonator_element, resonator_operation),
+    qubit=(qubit_element, qubit_operation),
+    integration_weights=int_w,
+    outputs=outs,
+)
+
+# Set calibrations from the ones detailed below
+my_calib.set_...
+
+# Open quantum machines manager
+qmm = QuantumMachinesManager(host="127.0.0.1", port=80)
+
+# Simulate the calibrations
+my_calib.simulate_calibrations(qmm, simulation_duration=5000)
```

Comment:
I would make it such that the `simulate_calibration` can accept both `qmm` and `qm`

Reply:
I thought that best practice is to use simulate with qmm, if I run it with qm I get a warning saying that I should use qmm...

Reply:
You're right, but since people are still using it, I would support both ways (just to make it easier for customers).

I also think we should remove the deprecated message, but this is for another topic

Reply:
We can discuss it, but if best practice is to use qmm (I don't know why actually), then we should update codes on the github that follow these best practices... Especially here, since they only have to write a single line of code `my_calib.simulate_calibrations(qmm, simulation_duration=5000)`

Reply:
I don't think there is a need to create two simulation functions, we can just make simulate_calibrations accept both a `QuantumMachinesManager` object and a `QuantumMachine` object. 
First line in the function can be "`if isinstance(machine, QuantumMachinesManager)`

We can also ditch this idea, it's not that critical, just thought it'll be nice.

Reply:
Ok done :)